pkg/openstack/common.go

```diff
@@ -248,25 +248,25 @@ func EnsureEndpointConfig(
 				if ingressOverride.TLS != nil && ingressOverride.TLS.SecretName != "" {
 					ed.Route.TLS.SecretName = ptr.To(ingressOverride.TLS.SecretName)
 					validateSecret := &tls.GenericService{SecretName: ed.Route.TLS.SecretName}
-					_, ctrlResult, err := validateSecret.ValidateCertSecret(ctx, helper, instance.GetNamespace())
+					_, err := validateSecret.ValidateCertSecret(ctx, helper, instance.GetNamespace())
 					if err != nil {
+						if k8s_errors.IsNotFound(err) {
+							instance.Status.Conditions.Set(condition.FalseCondition(
+								corev1.OpenStackControlPlaneCustomTLSReadyCondition,
+								condition.RequestedReason,
+								condition.SeverityInfo,
+								corev1.OpenStackControlPlaneCustomTLSReadyWaitingMessage,
+								ingressOverride.TLS.SecretName))
+							return endpoints, ctrl.Result{RequeueAfter: time.Duration(10)}, nil
```

Comment:
Does this duration need `* time.Second` as well?

Reply:
oh yes, thanks